### DISCUSSION TITLE
Cannot serve directory /var/www/html/: No matching DirectoryIndex (index.php,index.html)

### DISCUSSION THREAD

**Discussion by clockwiser** [docker-library/wordpress#650] [General]
After docker up, access to localhost returns error as below.
************************************************************
Forbidden
You don't have permission to access this resource.

Apache/2.4.48 (Debian) Server at localhost Port 8080
*************************************************************
Log details:
[Thu Oct 07 04:02:38.927782 2021] [autoindex:error] [pid 82] [client 172.20.0.1:49550] AH01276: Cannot serve directory /var/www/html/: No matching DirectoryIndex (index.php,index.html) found, and server-generated directory index forbidden by Options directive
*************************************************************

Problem is that index.php is in /var/www, not in /var/www/html.

Don't know why this happens.

**Comment by clockwiser** [comment]
Finally found the problem is caused by WORKDIR which was set to /var/www. Changing it to /var/www/html solved problem.
The last command is WORKDIR,  and I thought it won't affect the build and up process of docker.

**Comment by ladrua** [comment]
Ha same issue following these instructions for docker deployment example on docker hub page
```
FROM wordpress:apache
WORKDIR /usr/src/wordpress
RUN set -eux; \
	find /etc/apache2 -name '*.conf' -type f -exec sed -ri -e "s!/var/www/html!$PWD!g" -e "s!Directory /var/www/!Directory $PWD!g" '{}' +; \
	cp -s wp-config-docker.php wp-config.php
COPY custom-theme/ ./wp-content/themes/custom-theme/
COPY custom-plugin/ ./wp-content/plugins/custom-plugin/
```
where it tells me to change the workdir, should probably change back workdir or use full paths.
